app/components/concept-page/question-card.ts

```
@@ -75,14 +75,14 @@ export default class QuestionCardComponent extends Component<Signature> {
 
   @action
   handleDidInsertOptionsList(element: HTMLElement) {
```

ESLint

[error] 'element' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)